stackit/internal/services/iaas/network/resource.go

```diff
@@ -40,10 +41,19 @@ type Model struct {
 	NetworkId        types.String `tfsdk:"network_id"`
 	Name             types.String `tfsdk:"name"`
 	Nameservers      types.List   `tfsdk:"nameservers"`
+	IPv4Gateway      types.String `tfsdk:"ipv4_gateway"`
+	IPv4Nameservers  types.List   `tfsdk:"ipv4_nameservers"`
+	IPv4Prefix       types.String `tfsdk:"ipv4_prefix"`
 	IPv4PrefixLength types.Int64  `tfsdk:"ipv4_prefix_length"`
 	Prefixes         types.List   `tfsdk:"prefixes"`
+	IPv6Gateway      types.String `tfsdk:"ipv6_gateway"`
+	IPv6Nameservers  types.List   `tfsdk:"ipv6_nameservers"`
+	IPv6Prefix       types.String `tfsdk:"ipv6_prefix"`
+	IPv6PrefixLength types.Int64  `tfsdk:"ipv6_prefix_length"`
+	IPv6Prefixes     types.List   `tfsdk:"ipv6_prefixes"`
```

Comment:
Same for `IPv6Prefixes` and `IPv6Prefix`

Reply:
In the API design, there are two seperate fields.
Users can give a single prefix while creating a network. But in the response, there is a new field called prefixes where list of the prefixes are returned. 

same like ipv4 :)